### PR TITLE
Map newly added keras layers on python side

### PR DESCRIPTION
**Pull request by hkvision**
## What changes were proposed in this pull request?

Map newly added & merged keras layers on python side

## How was this patch tested?

Jenkins.

### CODE REVIEW

**Comment by hkvision** [comment]
@SeaOfOcean have a look at the updated highway?

File: spark/dl/src/test/scala/com/intel/analytics/bigdl/nn/HighwaySpec.scala [hunk 1]
```diff
@@ -42,7 +42,7 @@ class HighwaySpec extends KerasBaseSpec {
         |output_tensor = Highway(activation='tanh', bias=None)(input_tensor)
         |model = Model(input=input_tensor, output=output_tensor)
       """.stripMargin
-    val highway = Highway[Float](2, activation = "tanh", withBias = false)
+    val highway = Highway[Float](2, activation = Tanh[Float], withBias = false)
```
Comment (zhichao-li) [inline spark/dl/src/test/scala/com/intel/analytics/bigdl/nn/HighwaySpec.scala]: why change this?

Comment (hkvision) [reply spark/dl/src/test/scala/com/intel/analytics/bigdl/nn/HighwaySpec.scala]: Now using instance of activation on Scala side as other layers does.

**Comment by zhichao-li** [comment]
LGTM

**Comment by SeaOfOcean** [comment]
LGTM

**Comment by hkvision** [comment]
jenkins passed http://172.168.2.101:8080/view/PR-V/job/BigDL-PR-V/429/

**Pull request merged**